common/mariadb-galera/helm/templates/statefulset-mariadb.yaml

```diff
@@ -115,6 +115,31 @@ spec:
           privileged: {{ $.Values.initContainers.cleanoscache.securityContext.privileged }}
           runAsUser: {{ $.Values.initContainers.cleanoscache.securityContext.runAsUser | default 0 | int }}
       {{- end }}
+      - name: fix-mariadb-fs-permissions
```

Comment:
Please make that initContainer optional like we have done it for the [clean-os-cache](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/templates/statefulset-mariadb.yaml#L106) container. Please also create a [mariab.fixFsPermissions](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/values.yaml#L66) boolean option (default false) and add some documentation similar to [wipeDataAndLog](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/values.yaml#L69-L71)

Reply:
done!